src/components/Export.tsx

```diff
@@ -316,21 +321,48 @@ const ExportDialog: ExportDialogComponent = ({
     if (typeof results === "object") {
       window.roamAlphaAPI.ui.graphView.wholeGraph.setMode("Explore");
       window.roamAlphaAPI.ui.graphView.wholeGraph.setExplorePages(
-        livePages.map((r) => r.text)
+        livePages.map((r) => String(r[firstColumnKey]))
       );
       window.location.href = `${getRoamUrl()}/graph`;
     }
   };
 
-  const handleSendTo = () => {
-    if (isSendToGraph) {
-      addToGraphOverView();
-    } else if (isCanvasPage) {
-      addToSelectedCanvas();
-    } else {
-      addToSelectedPage();
+  const handleSendTo = async () => {
+    try {
+      if (isSendToGraph) addToGraphOverView();
+      else if (isCanvasPage) await addToSelectedCanvas();
+      else addToSelectedPage();
+      renderToast({
+        content: "Results sent!",
+        intent: "success",
+        id: "query-builder-export-success",
+      });
+    } catch (e) {
+      const error = e as Error;
+      renderToast({
+        content: "Looks like there was an error.  The team has been notified.",
```

Comment:
[nit] looks like an unintentional extra space here

Reply:
hah!  I'm still in the 'two spaces after a period' era apparently. Muscle memory can be a bitch.